CMakeLists.txt

```diff
@@ -151,9 +151,10 @@ if(BORINGSSL_PREFIX AND BORINGSSL_PREFIX_SYMBOLS AND GO_EXECUTABLE)
     COMMAND ${GO_EXECUTABLE} run ${CMAKE_CURRENT_SOURCE_DIR}/util/make_prefix_headers.go -out ${CMAKE_CURRENT_BINARY_DIR}/symbol_prefix_include ${BORINGSSL_PREFIX_SYMBOLS_PATH}
     # Temporary exclude all Kyber symbols.
     # TODO(dkostic): fix the prefix build to work with Kyber source code.
-    COMMAND sed -i '/kyber/d' ${CMAKE_CURRENT_BINARY_DIR}/symbol_prefix_include/boringssl_prefix_symbols.h
-    COMMAND sed -i '/kyber/d' ${CMAKE_CURRENT_BINARY_DIR}/symbol_prefix_include/boringssl_prefix_symbols_asm.h
-    COMMAND sed -i '/kyber/d' ${CMAKE_CURRENT_BINARY_DIR}/symbol_prefix_include/boringssl_prefix_symbols_nasm.inc
+    COMMAND sed -i.bak '/pqcrystals/d' ${CMAKE_CURRENT_BINARY_DIR}/symbol_prefix_include/boringssl_prefix_symbols.h
```

Comment:
I missed that this line was added. I'm totally confused now. The PR description says:
```
- Update headers for Kyber to support prefixing.
```
So if we support prefixing Kyber symbols now, why do we have to exclude them with this `sed`?

Reply:
After a prefix build, all "kyber" symbols are prefixed -- however, "bignum" symbols are not prefixed.  So they both are being removed from `boringssl_prefix_symbols.h` but for different reasons. 

I didn't remove the "pqcrystals" from the assembly headers b/c I assumed prefix-adding macros would still be needed there if the functions were being called from assembly code.